disnake/guild.py

```diff
@@ -2106,6 +2202,30 @@ async def edit(
 
             .. versionadded:: 2.6
 
+        invites_disabled_until: Optional[Union[:class:`datetime.datetime`, :class:`datetime.timedelta`]]
+            The time until/for which invites are paused.
+            See also the ``invites_disabled`` parameter.
+
+            Can be set to ``None`` to re-enable invites.
+
+            This is only available to guilds that contain ``COMMUNITY``
+            in :attr:`Guild.features`.
+
+            .. versionadded:: 2.10
+
+        dms_disabled_until: Union[:class:`datetime.datetime`, :class:`datetime.timedelta`]
+            The time until/for which DMs between guild members are disabled.
+
+            This does not apply to moderators, bots, or members who are
+            already friends with each other.
+
+            Can be set to ``None`` to re-enable DMs.
+
+            This is only available to guilds that contain ``COMMUNITY``
+            in :attr:`Guild.features`.
+
+            .. versionadded:: 2.10
+
```

Comment:
These both can only be enabled for a maximal timestamp of 24 hours in the future, it would be great adding a note mentioning that (just as `timeout` in `Member.edit`)